Fix #21

brayniverse opened the pull request:
This hotfix worked for me, however I can't speak for the majority.

Commit:
Fix #21

brettshollenberger commented:
@brayniverse - Currently the test suite is failing with this change. Can you write failing test for this, and make these others pass?

Commit:
Update regex to find port number (#21)

Commit:
More consistent solution

This `\:[a-zA-Z_]+` regex pattern is used elsewhere in this module to perform the same task. Thought it would be better to maintain a level of consistency. All tests passed with this change.

brayniverse commented:
Hey @brettshollenberger, I did a bit more digging around to see if I was just covering up a bug elsewhere and found you were using this regular expression `\:[a-zA-Z_]+` elsewhere and thought it was a better idea to modify this PR to be more consistent with the rest of the module. What do you think?

brettshollenberger commented:
@brayniverse I updated that regex and fixed on 0.7.4. Thanks for finding this!
